creator/ingest_runs/manifests.py

```diff
@@ -1,334 +0,0 @@
-        ast.literal_eval(key)['external_id']: value
-        for key, value in genomic_cache[GEN_FILE].items()
-    }
-    hash_df = pd.DataFrame(
-        list(cache_dict.items()),
-        columns=[
-            CONCEPT.GENOMIC_FILE.ID, CONCEPT.GENOMIC_FILE.TARGET_SERVICE_ID
-        ],
-    )
-    genomic_df = genomic_df.merge(
-        right=hash_df,
-        on=CONCEPT.GENOMIC_FILE.ID,
-        how='inner',
-    )
-    return genomic_df
-
-
-def load_entities(entity_type, df, study_id):
-    """
-    Load data of type _entity_type_ into the DataService from _df_.
-    """
-    target_service_base_url = LOCALHOST
-    app_settings = settings.load()
-    loader = LoadStage(
-        app_settings.TARGET_API_CONFIG,
-        target_service_base_url,
-        [entity_type],
-        study_id,
-        cache_dir=os.getcwd(),
-    )
-    loader.run({entity_type: df})
-
-    return loader.uid_cache
-
-
-def munge_genomic_df(df):
-    """
-    Clean up the genomic-file DataFrame and get it ready to be ingested by
-    removing unneeded columns and standardizing the column names.
-    """
-    df['Hashes'] = df.apply(lambda row: {"ETag": row['ETag']}, axis=1)
-    df['urls'] = df.apply(lambda row: [row['Filepath']], axis=1)
-    df['file_type'] = df.apply(
-        lambda row: os.path.splitext(row['Filename'])[1][1:],
-        axis=1,
-    )
-    col_rename = {
-        'KF Biospecimen ID': CONCEPT.BIOSPECIMEN.ID,
-        'Data Type': CONCEPT.GENOMIC_FILE.DATA_TYPE,
-        'Filename': CONCEPT.GENOMIC_FILE.FILE_NAME,
-        'Hashes': CONCEPT.GENOMIC_FILE.HASH_DICT,
-        'Size': CONCEPT.GENOMIC_FILE.SIZE,
-        'urls': CONCEPT.GENOMIC_FILE.URL_LIST,
-        'file_type': CONCEPT.GENOMIC_FILE.FILE_FORMAT,
-        'Filepath': CONCEPT.GENOMIC_FILE.ID,
-    }
-    df = df[list(col_rename.keys())]
-    df.rename(columns=col_rename, inplace=True)
-    df[CONCEPT.GENOMIC_FILE.HARMONIZED] = True
-    return df
-
-
-def get_biospecimen_sequencing_experiment_links(study_id):
-    """
-    Get the Biospecimen-Sequencing Experiment links by performing the following
-    steps:
-    1. Get the study's unharmonized genomic files
-    2. Get the study's sequencing experiments
-    3. Get the study's sequencing-experiment-genomic-file links
-    4. Get the study's biospecimen-genomic-file links
-    5. Join the tables from 1, 2, 3, 4 together on the genomic file ID
-    Returns the table from step 5.
-    """
-    dfs = {}
-    entity_types = {
-        GEN_FILES,
-        SEQ_EXPS,
-        SEQ_EXP_GEN_FILES,
-        BIO_GEN_FILES,
-    }
-
-    # Steps 1-4
-    for entity_type in entity_types:
-        dfs[entity_type] = get_entities(entity_type, study_id)
-
-    # Step 5. Join tables 1-4 on genomic file ID. Some preprocessing needs to
-    # be done first.
-    
-    # Start by merging the DataFrames from steps 1 and 3 on genomic-file ID
-    GEN_LINK = '_links.genomic_file'
-
-    # Check the validity of the SEQ_EXP_GEN_FILES DataFrame
```

Comment:
I removed these assertions because I think it's safe to assume the output of the kf_utils.yield_entities method won't change.